docs/admin/install.rst

```diff
@@ -117,19 +115,42 @@ Follow the `installation documentation <https://www.qubes-os.org/doc/installatio
 - Use all available storage space for the installation (as the computer should be dedicated to SecureDrop Workstation).
 - Set a strong FDE passphrase - a 6-word Diceware passphrase is recommended.
 - Create an administrative account named ``user`` with a strong password.
+- In the template configuration, ensure that the following options are checked:
+
+  - "Create default system qubes (sys-net, sys-firewall, default DispVM)"
+  - "Make sys-firewall and sys-usb disposable"
+- If there is a grayed out option "USB qube configuration disabled", make a note of this. An additional setup step will be required (see below).
 
-  .. note:: Qubes is not intended to have multiple user accounts, so this account name and password will be shared by all SecureDrop Workstation users. The password will be required to log in and unlock the screen during sessions - choosing something strong but memorable and easily typed is recommended!
+  .. note:: Qubes is not intended to have multiple user accounts, so your account name and password will be shared by all SecureDrop Workstation users. The password will be required to log in and unlock the screen during sessions - choosing something strong but memorable and easily typed is recommended!
 
 Once the installation is complete, you will be prompted to reboot into Qubes. Reboot, removing the install USB when the computer restarts.
 
 You will be prompted to enter the FDE passphrase set during installation.
 
-.. note:: On first booting into a Qubes OS 4.0.4 installation, you may be prompted to enter the FDE passphrase at a command-line prompt rather than via the GUI. The next system update will restore the GUI prompt.
-
 After the disk is unlocked and Qubes starts, you will be prompted to complete the initial setup. Click the Qubes OS icon, then accept the default options and click **Done**. Finally, click **Finish Configuration** to set up the default system TemplateVMs and AppVMs.
 
 Once the initial setup is complete, the login dialog will be displayed. Log in using the username and password set during installation.
 
+If, during the installation, you encountered the grayed out option "USB qube configuration disabled", you must now create a VM to access your USB devices. To do so, open a ``dom0`` terminal via the Qubes menu (the **Q** icon in the upper left corner): **Q > Terminal Emulator**. Run the following command:
+
+.. code-block:: sh
+
+  sudo qubesctl state.sls qvm.sys-usb
+
+After the command exits, confirm that you see an entry "Service: sys-usb" in the Qubes menu.
+
+Apply ``dom0`` updates (estimated wait time: 15-30 minutes)
```

Comment:
For 4.1.0 in particular, this is critical due to the broken update check logic. This should no longer be an issue on 4.1.1, but it still seems to me that having `qubes-dom0-update` as an independent step makes sense, just in case of any QSBs since the stable release.